### DISCUSSION TITLE
Log in node js

### DISCUSSION THREAD

**Discussion by EduardoNoronha31** [nodejs/help#5027] [Q&A] [answered]
how log a message when init a project with node js and express?

**Comment by gusttavonl** [comment, marked as answer]
my suggestion is here https://github.com/nodejs/node/discussions/46260#discussioncomment-4721636, in my example has a line with this code 

```
  app.listen(3000, () =>
      console.log(`The server is started!`)
    );
```

the listen is the start from your server
